modules/data_dictionary_widget/data_dictionary_widget.module

```diff
@@ -76,7 +76,7 @@ function data_dictionary_widget__data_dictionary_data_type_checker($context) {
 function data_dictionary_widget_form_alter(&$form, &$form_state, $form_id) {
   $formObject = $form_state->getFormObject();
 
-  if ($formObject instanceof \Drupal\Core\Entity\EntityFormInterface) {
+  if ($formObject instanceof \Drupal\Core\Entity\EntityFormInterface && $form_id == 'node_data_form') {
```

Comment:
Change this to   `if ($formObject instanceof \Drupal\Core\Entity\EntityFormInterface & in_array($form_id, $target_form_ids)) {
`